data/human/human missions.txt

```diff
@@ -4965,3 +4965,183 @@ mission "Riley"
+			choice
+				`	"(Go up to the sword.)"`
+						goto sword-scene
+				`	"(Take a look around first.)"`
+			`	You walk around, looking at the various alien artifacts. "Wow, he must have always brought us back the small items from his expeditions, because these artifacts over here are huge!" Liam exclaims from another part of the dome. "How did he even move all these here?"`
+			`	Now that Liam points it out, many of the larger artifacts look similar to the Sheragi technology you studied with Albert Foster.`
+				to display
+					has "Sheragi Archaeology: The Emerald Sword 3: done"
+			`	After some time, Sophie calls you both to come to look at the tablet.`
+			label sword-scene
+			scene "scene/sword dome"
+			`	Approaching the tablet, there is a short message written on it:`
+			`	"My dear children. The galaxy runs far and wide, and I have traveled many places, more than most in many lifetimes. I have seen many worlds, both alien and human, too. And it was on those worlds that I have been exploring, discovering, and researching everything you see around you. It was my life's duty.`
+			`	But still, there were many paths I did not take, many rocks I have left unturned, and many worlds I have not yet landed on. And I suspect that you will have read this before those destinations were reached. Well, I have arrived at a different destination now, the only one nobody can understand, but everybody reaches eventually. Do not fear for me.`
+			`	This room you are in now is the culmination of all my work. I hope you understand that it was for your sake that I kept these a secret. My journey was long and unforgiving, but it was my life. I hope you continue that journey, to understand the universe a little better.`
+			`	Your father."`
+			`	You leave the two of them alone for a bit, walking around and looking at the artifacts.`
+			`	After some time they call you to them. Sophie starts, "Captain <last>, Liam and I have thought through about this, and we think it would be a shame if we just kept this sword as a memory to him. It needs to be used. It was our father's one and only weapon, and it has been many places. Take it to many more." She pulls the sword from the pedestal and hands it to you. "It is yours now."`
+			choice
+				`	"I would be more than grateful to accept it."`
+						goto accepted
+				`	"This is too much for me to take."`
+						goto declined
+			label declined
+			`	"Are you sure?" Liam asks. "This is a one time offer. It would be wasted on us."`
+			choice
+				`	"Yes, I'm sure."`
+				`	"Well, if you frame it that way, I will take it."`
+					goto accepted
+			`	"I see. Please, at least take this extra payment of <credits>," Liam says. "It is for you traveling so much with us."`
+				action
+					payment 50000
+				goto end
+			label accept-sword
+			`	"Please take it on some more adventures," Sophie says.`
```

Comment:
```suggestion
			label accepted
```
I have to remove this line because with the new dialogue, Sophie repeats herself now. I might add a new piece of dialogue here for the player to try out the sword like it had before.